components/core/config/schemas.txt

```diff
@@ -1,19 +1,98 @@
-// Delimiters
-delimiters: \t\r\n!"#$%&'\(\)\*,:;<>?@\[\]\^_`\{\|\}~
+// ===
+// Timestamps
+// ===
+
+// E.g. 2015-01-31T15:50:45.392
+timestamp:\d{4}\-\d{2}\-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}
```

Comment:
Everywhere we use `T`, technically we could also support a space. Same for periods and commas. So:

```suggestion
// E.g. 2015-01-31T15:50:45.392
// E.g. 2015-01-31T15:50:45,392
// E.g. 2015-01-31 15:50:45.392
// E.g. 2015-01-31 15:50:45,392
timestamp:\d{4}\-\d{2}\-\d{2}[T ]\d{2}:\d{2}:\d{2}[,\.]\d{3}
```

Unless that causes any issues?